packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.test.tsx

```diff
@@ -97,33 +98,29 @@ async function rendersAsExpected(
       ? getLocalizedDateString(invoicePreview.line_items[0].period.end)
       : getLocalizedDateString(customerWebSubscription.current_period_end);
 
+  const formattedInterval = formatPlanInterval({
+    interval: selectedPlan.interval,
+    intervalCount: 1,
+  });
+
```

Comment:
This can be removed.

`selectedPlan.interval` in this case should return the already formatted interval.